### PR TITLE
Authorize with service account

### PR DESCRIPTION
**Pull request by Haziba**
**Notes to tester:**
Relies on PR #13

### CODE REVIEW

File: test/Cronofy.Test/CronofyEnterpriseConnectAccountClientTests/AuthorizeUser.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using NUnit.Framework;
+
+namespace Cronofy.Test.CronofyEnterpriseConnectAccountClientTests
+{
+    internal sealed class AuthorizeUser : Base
+    {
+        [Test]
+        public void CanAuthorizeUser()
```
Comment (stephenbinns) [inline test/Cronofy.Test/CronofyEnterpriseConnectAccountClientTests/AuthorizeUser.cs]: Do you want to add another test for the passing enumerable version?

File: src/Cronofy/CronofyEnterpriseConnectAccountClient.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+            AuthorizeUser(email, callbackUrl, String.Join(" ", scope.ToArray()));
+        }
+
+        /// <inheritdoc/>
+        public void AuthorizeUser(string email, string callbackUrl, string scope)
+        {
+            Preconditions.NotNull("email", email);
+            Preconditions.NotNull("callbackUrl", callbackUrl);
+            Preconditions.NotNull("scope", scope);
```
Comment (stephenbinns) [inline src/Cronofy/CronofyEnterpriseConnectAccountClient.cs]: I think you can be more specific with these and say NotBlank

File: src/Cronofy/CronofyEnterpriseConnectAccountClient.cs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+            return response.Resources.Select(x => x.ToResource());
+        }
+
+        /// <inheritdoc/>
+        public void AuthorizeUser(string email, string callbackUrl, IEnumerable<string> scope)
+        {
+            Preconditions.NotNull("email", email);
+            Preconditions.NotNull("callbackUrl", callbackUrl);
+            Preconditions.NotNull("scope", scope);
```
Comment (stephenbinns) [inline src/Cronofy/CronofyEnterpriseConnectAccountClient.cs]: These asserts are unneeded as you're delegating to the other method

**Comment by Haziba** [comment]
@stephenbinns :mag:

**Comment by stephenbinns** [comment]
👍

**Pull request merged**